### PR TITLE
script: fixing shell errors for shellcheck

### PR DESCRIPTION
**Pull request by outscale-avu**
Signed-off-by: Axel Viau <axel.viau@outscale.com>

### CODE REVIEW

File: dpdk_symbols.sh [hunk 1]
```diff
@@ -10,20 +10,20 @@ tmp=pg_tmp
...
     fi
 done
 
 echo -n "" > dpdk_symbols_autogen.h
 
-cat $tmp | while read f; do
+while read f; do
     echo "PG_DPDK_INIT($f)" >> dpdk_symbols_autogen.h
-done
+done < "$tmp"
```
Comment (outscale-avu) [inline dpdk_symbols.sh]: Avoid useless cat : [https://github.com/koalaman/shellcheck/wiki/SC2002](url)

File: dpdk_symbols.sh [hunk 2]
```diff
@@ -10,20 +10,20 @@ tmp=pg_tmp
...
-        objcopy --globalize-symbols=$tmp $a
+        objcopy --globalize-symbols=$tmp "$a"
     fi
 done
 
 echo -n "" > dpdk_symbols_autogen.h
 
-cat $tmp | while read f; do
+while read f; do
     echo "PG_DPDK_INIT($f)" >> dpdk_symbols_autogen.h
```
Comment (outscale-avu) [inline dpdk_symbols.sh]: Avoid useless cat : [https://github.com/koalaman/shellcheck/wiki/SC2002](url)

File: dpdk_symbols.sh (hunk 1 above)
Comment (outscale-avu) [inline dpdk_symbols.sh]: Avoid useless cat : [Shellcheck Wiki](https://github.com/koalaman/shellcheck/wiki/SC2002)

File: examples/switch/run_vhost.sh [hunk 3]
```diff
@@ -84,34 +84,34 @@ function qemu_start {
...
             echo "Still bad md5 for $path ... abort."
             exit 1
         fi
     fi
 }
 
 function check_bin {
-    run=${@:1}
-    $run &> /dev/null
+	run=( "${@:1}" )
```
Comment (outscale-toa) [inline examples/switch/run_vhost.sh]: can you remove the white space please ?

File: examples/switch/run_vhost.sh [hunk 4]
```diff
@@ -8,30 +8,32 @@ declare -A qemu_pids
...
+	cmd=( "${@:3}" )
+	# shellcheck disable=SC2029
+    ssh -q -p 500"$id" -l root -i vm.rsa -oStrictHostKeyChecking=no -oConnectTimeout="$timeout" 127.0.0.1 "${cmd[@]}"
 }
 
 function ssh_bash {
     id=$1
-    cmd="${@:2}"
-    ssh -p 500$id -l root -i vma.rsa -oStrictHostKeyChecking=no 127.0.0.1
+	cmd=( "${@:2}" )
```
Comment (outscale-toa) [inline examples/switch/run_vhost.sh]: can you remove the white space please ?
I thinks you can remove cmd

**Pull request merged**